_src/static/sass/components/_copy.scss

```diff
@@ -0,0 +1,118 @@
+// ------------------------------
+// edX Pattern Library: Components - Copy
+
+// About: Contains base styling for copy (general text and type used in content and functionaly).
+
+// #GLOBAL
+// #INDIVIDUAL COPY CASES
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.copy {
+    @extend %reset-copy;
```

Comment:
Yeah, I'll let the cascade of our base/reset styles shine through here and simplify what I'm extending and defining in each case.